### PR TITLE
feat(fast-jss-utilities): add a utilitiy to convert hex values to rgb(a)

### LINKED ISSUE #639: Add a utility to convert hex values to rgb or rgba
Add a utility to convert hex values to either RGB or RGBA color values.

Users should be able to pass design system values with an opacity and return a valid color.

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>
closes #639 
<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-jss-utilities/src/color.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export function convertHexToRgb(hex: string, alpha?: number): string {
```
Comment (nalogiudice) [inline packages/fast-jss-utilities/src/color.ts]: nit: Just a thought - why not just `hexToRgb` I don't think any understanding is lost and it is shorter.

Comment (nicholasrice) [inline packages/fast-jss-utilities/src/color.ts]: Part of me thinks that we shouldn't expose the alpha logic in this function. It doesn't really seem part of the scope of converting hex to rgb. 

Also, chroma already does color conversion - which we take a dependency on. Is there a reason we aren't just using chromas utilities?

Comment (chrisdholt) [reply packages/fast-jss-utilities/src/color.ts]: At some point I think there was a conversation that over time we want to reduce our dependency on Chroma where we can for smaller things like this. We can't tree shake it, so if we just wanted this functionality, we are still taking the whole thing.

If that's not a huge concern, I'll just abandon this. 

There is currently an issue on chroma for converting to ES6 to support this, but it's been open for over a year. We could always contribute it in

Comment (nicholasrice) [reply packages/fast-jss-utilities/src/color.ts]: Yea on one hand adding this now reduces the amount of work we'ed need to do later. On the other, it adds code in the interim and moving away from chroma isn't super high priority at the moment. 

I think i'm in favor of just using chroma for now and doing a migration if/when it makes sense, but I'm open to other's thoughts on the matter.

Comment (chrisdholt) [reply packages/fast-jss-utilities/src/color.ts]: I'm fine with this. I'll just close it out.

**Comment by chrisdholt** [comment]
Closing as moving from Chroma is not a high pri and there is more work to do to fully address it.

**Pull request closed without merging**